src/common/acl_util.c

```diff
@@ -390,6 +390,10 @@ perms_unified(struct daos_ace *ace)
 	return true;
 }
 
+#define	WRITE_CH_FOR_BIT(bits, bit_name)				\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon